cypress/e2e/inventory/item/delete-an-item-without-dependencies.cy.js

```diff
@@ -0,0 +1,85 @@
+import uuid from 'uuid';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import QuickMarcEditor from '../../../support/fragments/quickMarcEditor';
+import TopMenu from '../../../support/fragments/topMenu';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import { JOB_STATUS_NAMES, LOAN_TYPE_NAMES, MATERIAL_TYPE_NAMES } from '../../../support/constants';
+import ItemRecordNew from '../../../support/fragments/inventory/item/itemRecordNew';
+import HoldingsRecordView from '../../../support/fragments/inventory/holdingsRecordView';
+import InventoryItems from '../../../support/fragments/inventory/item/inventoryItems';
+
+describe('Importing MARC Bib files', () => {
+  const testData = {
+    tag852: '852',
+    headerTitle: 'Create a new MARC Holdings record',
+    location: QuickMarcEditor.getExistingLocation(),
+    updatedIndicator: '0',
+    newTag852Content: '$b E $h call $i number $k prefix $l title $m suffix $t copy number',
+    itemBarcode: uuid(),
+    itemMaterialType: MATERIAL_TYPE_NAMES.ELECTRONIC_RESOURCE,
+    itemLoanType: LOAN_TYPE_NAMES.CAN_CIRCULATE,
+  };
+  const marcFile = {
+    marc: 'oneMarcBib.mrc',
+    fileName: `testMarcFileC715${getRandomPostfix()}.mrc`,
+    jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+  };
+  const recordIDs = [];
+
+  before('create test data and login', () => {
+    cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(() => {
```

Comment:
Please don't use Data import in this case.
It would be better to use InventoryInstances.createFolioInstanceViaApi() function